tests/commands/__init__.py

```diff
@@ -0,0 +1,69 @@
+import os
+import sys
+import unittest
+import importlib
+from textwrap import dedent
+from pykickstart.base import KickstartCommand, BaseData
+
+class ClassDefinitionTestCase(unittest.TestCase):
+    """
+        Search for command and data classes defined like so:
+
+        RHEL7_AutoPart = F21_AutoPart.
+
+        This kind of definitions makes it possible for other
+        tests to omit possible errors and we don't like to have
+        them. Either we use the existing class name if we haven't
+        redefined anything or provide a boilerplate definition:
+
+        class RHEL7_Autopart(F21_Autopart):
+            pass
+    """
+    def runTest(self):
+        errors = 0
+        commands_dir = os.path.join(os.path.dirname(__file__), "..", "..", "pykickstart", "commands")
+        commands_dir = os.path.abspath(commands_dir)
+
+        self.assertTrue(os.path.exists(commands_dir))
+        if commands_dir not in sys.path:
+            sys.path.append(commands_dir)
+
+        for dirpath, _dirnames, paths in os.walk(commands_dir):
+            for path in paths:
+                if (path == '__init__.py') or \
+                    not path.endswith('.py'):
```

Comment:
These two can be one line, without the extra parens.  That's easier to read as far as I'm concerned.